ex7/index.js

```diff
@@ -0,0 +1,19 @@
+const prompt = require("prompt-sync")({ sigint: true});
+
+console.log("Insert the number of apples purchased:");
+const apples = Number(prompt(">"));
+
+function shop() {
+    if (apples < 12) {
+        let applesLess = (apples * 0.30);
+        applesLess = applesLess.toFixed(2);
+        return (`Your purchase will cost $${applesLess} dollars.`);
+    } 
+    if (apples >= 12) {
+        let applesMore = (apples * 0.25);
```

Comment:
We usually want to avoid **_Magic Numbers_** ✨. Please consider moving the numbers `0.3` and `0.25` to properly named constants.

```js
const DEFAULT_PRICE = 0.3
const DISCOUNT_PRICE = 0.25
```